docs/configuring.rst

```diff
@@ -1,108 +1,210 @@
+Theme options
+=============
 
-HTML Theme Options
-------------------
+The following options can be defined in your project's ``conf.py`` file, using
+the :confval:`html_theme_options <sphinx:html_theme_options>` configuration option.
 
-The theme's project-wide options are defined in the ``sphinx_rtd_theme/theme.conf``
-file of this repository, and can be defined in your project's ``conf.py`` via
-``html_theme_options``. For example:
+For example:
 
 .. code:: python
 
     html_theme_options = {
-        'canonical_url': '',
+        'canonical_url': 'https://example.com/en/latest/',
         'analytics_id': '',
         'logo_only': False,
         'display_version': True,
         'prev_next_buttons_location': 'bottom',
         'style_external_links': False,
         'vcs_pageview_mode': '',
-        # Toc options
         'collapse_navigation': True,
         'sticky_navigation': True,
         'navigation_depth': 4,
         'includehidden': True,
         'titles_only': False
     }
 
-The following options are available:
+Table of contents options
+-------------------------
+
+The following options affect the `table of contents configuration options`_
+Sphinx uses to generate the documentation navigation.
+
+.. confval:: collapse_navigation
+
+    :type: boolean
+    :default: ``False``
+
+    With this enabled, navigation entries are not expandable -- the ``[+]``
+    icons next to each entry are removed.
+
+.. confval:: sticky_navigation
+
+    :type: boolean
+    :default: ``True``
+
+    Scroll the with the main page content as you scroll the page.
+
+.. confval:: navigation_depth
+
+    :type: integer
+    :default: ``4``
+
+    The maximum depth of the table of contents tree. Set
+    this to ``-1`` to allow unlimited depth
 
-Base options
-~~~~~~~~~~~~
+.. confval:: includehidden
 
-* ``canonical_url`` String. This will specify a `canonical url <https://en.wikipedia.org/wiki/Canonical_link_element>`__
-  to let search engines know they should give higher ranking to latest version of the docs.
-  The url points to the root of the documentation and requires a trailing slash.
-* ``analytics_id`` String. Change the Google Analytics ID that is included on pages.
-* ``display_version`` Bool. With this disabled, the version number isn't shown at the top of the sidebar.
-* ``prev_next_buttons_location`` String. can take the value ``bottom``, ``top``, ``both`` , or ``None``
-  and will display the "Next" and "Previous" buttons accordingly.
-* ``style_external_links`` Bool. Add an icon next to external links. Defaults to ``False``.
-* ``vcs_pageview_mode`` String. Changes how to view files when using `display_github`, `display_gitlab`, etc.
-  When using Github or Gitlab this can be: `blob` (default), `edit`, or `raw`,
-  on Bitbucket, this can be either: `view` (default) or `edit`.
+    :type: boolean
+    :default: ``True``
 
-TOC Options
-~~~~~~~~~~~
+    Specifies if the navigation includes TOC trees marked with
```

Comment:
"TOC trees" is Sphinx terminology that you wanted to avoid. You could use `...table(s) of contents marked with...` or refer to the actual rst role: `...includes ``toctree``s marked with...`